generative_ai/embeddings/document_retrieval_example.py

```diff
@@ -35,12 +35,18 @@ def embed_text() -> list[list[float]]:
     model = TextEmbeddingModel.from_pretrained("gemini-embedding-001")
     inputs = [TextEmbeddingInput(text, task) for text in texts]
```

Comment:
This variable is unused and is failing linting. 
You're constructing `text_input` later for each item in the for loop, so this variable should be removed.

Reply:
Done